product_lifecycle/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Product Lifecycle",
+    "summary": "Manage replacement of obsolete products",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "product",
+    "depends": [
+        "sale_stock",
+    ],
+    "data": [
+        "data/ir_cron.xml",
+        "views/product_view.xml",
+        "views/sale_order_views.xml",
+        "views/assets.xml",
+    ],
+    "demo": [
+        "demo/replacement_product_demo.xml",
+    ],
+    "qweb": [
+        "static/src/xml/replacement_template.xml",
+    ],
+    "installable": True,
```

Comment:
Could you add my user as maintainer, please?

Reply:
Done. Could you check this MR?